tests/features/markdown.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Markdown
+    The journal accepts Markdown syntax in entries and renders them properly.
+
+Scenario: An authorized user can use Markdown in an entry
+    Given an authenticated user
+    When the user creates an entry
+    Then they can use Markdown to format the entry
```

Comment:
I would have implemented this step as a `when`, so that you can have a `then` that says something like "then the user can see the text formatted properly"